docs/cryoet_data_portal_docsite_FAQ.md

```diff
@@ -1,4 +1,209 @@
+This allows our team to follow up with you on Github if we have a question about the problem you encountered. Then, [fill out this form](https://github.com/chanzuckerberg/cryoet-data-portal/issues/new). 
+We suggest you use a descriptive title, paste an error messages using the `<>` icon on the form, and provide as many details as possible about the problem, including what you expected to happen and what type of machine you were using.
+
+An example issue is below:
+
+Title: Tomogram TS_026 cannot be downloaded
+
+Body: 
+I have the AWS CLI tool installed on a Mac computer. I copied the download command from the prompt on the tomogram page. Instead of downloading, I received this error message:
+```
+ERROR MESSAGE COPIED FROM TERMINAL
+```
+------------------------------------------
+For more information about submiting issues on Github, please refer to [Github's documentation](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-an-issue#creating-an-issue-from-a-repository).
+
+## What are datasets, runs, and annotations in the Data Portal?
+
+The CryoET Data Portal uses the following data schema:
+
+1. A dataset is a community contributed set of image files for tilt series, reconstructed tomograms, and if available, cellular and/or subcellular annotation files. Every dataset contains only one sample type prepared and imaged with the same conditions. The dataset title, such as `S. pombe cryo-FIB lamellae acquired with defocus-only`, summarizes these conditions. Samples can be a cell, tissue or organism; intact organelle; in-vitro mixture of macromolecules or their complex; or in-silico synthetic data, where the experimental conditions are kept constant. Downloading a dataset downloads all files, including all available tilt series, tomograms, and annotations.
+2. A run is one experiment, or replicate, associated with a dataset, where all runs in a dataset have the same sample and imaging conditions. Every run contains a collection of all tomography data and annotations related to imaging one physical location in a sample. It typically contains one tilt series and all associated data (e.g. movie frames, tilt series image stack, tomograms, annotations, and metadata), but in some cases, it may be a set of tilt series that form a mosaic. When downloading a run, you may download only the tomograms or the tomograms and available annotations.
+3. An annotation is a point or segmentation indicating the location of a macromolecular complex in the tomogram. On the run page, you may choose to download tomograms with their annotations.
+
+Descriptions of all terminology and metadata used in the Portal is provided [here](https://docs.google.com/document/d/11h0u3YYF1EWCTjxu3ObShx26HgLAfJhn9I_tIaeQ6GI/edit#?usp=sharing). You can refer to a graphic of the [data schema here](https://chanzuckerberg.github.io/cryoet-data-portal/python-api.html#data-model).
```

Comment:
the latest version of the metadata is located here: https://docs.google.com/document/d/1zcD9OyY86gaogVZOBZPwcn0fxZ5QSFrlqXeaodJX9BI/edit#heading=h.be2fyivqz1zp

although I'm not sure if the metadata google doc is meant to be shared to the wider audience @anchi2c could you help confirm?

Reply:
The link is connected to the working copy.  In the least a copy of it should be made so that the world can see it instead of restricted to CZI.  We could convert it into PDF if we don't want outside users to comment on it.  The former was what we did in phase 1 release.  I am not familiar of using Google doc this way, so I think @junxini and @EricMaxWang are better judge of this.